portal-kernel/test/unit/com/liferay/portal/kernel/util/ValidatorTest.java

```diff
@@ -175,7 +175,7 @@ public void testIsInvalidHostName() throws Exception {
 	public void testIsInvalidIPv4Address() throws Exception {
 		testValidIPv4Addresses(
 			new String[] {
-				"392.168.1.102", "255.0.0", "256.257.258.259",
+				null, "392.168.1.102", "255.0.0", "256.257.258.259",
 				"128.0000.001.002", "10.10.10.1000", "0192.0168.0001.0001"
```

Comment:
@alejandrozarza, please add String Blank check in the testIsInvalidIPv4Address(). This will keep consistent with testIsInvalidIPv6Address(). You may git push force the change. After that, I will approve it. Thanks.